analysis/bin/vtsAnalyser.cc

```diff
@@ -25,10 +25,21 @@ string getType (const string& path){
   size_t found = path.find("tt");
   return(path.substr(found));
 }
-
+long getFileSize(const string& filename) {
+  struct stat st;
+  if ( stat(filename.c_str(), &st) == 0) {
+    return st.st_size;
+  } else return 0;  
+}
+long getFileSize(const char* filename) {
+  struct stat st;
+  if ( stat(filename, &st) == 0) {
+    return st.st_size;
+  } else return 0;
+}
 
 int main(Int_t argc, Char_t** argv) {
-
+//int main(int argc, char* argv[]) {
```

Comment:
These two lines are the same, the commented out one is the usual way to write it in C.